src/staking-router/staking-router.service.ts

```diff
@@ -13,6 +15,77 @@ export class StakingRouterService {
     protected readonly keysApiService: KeysApiService,
   ) {}
 
+  public async getVettedAndUnusedKeys() {
```

Comment:
I suggest making the endpoint on the Keys-API side. it can be done in SQL using sorting and $lt $gt operators

Reply:
i am okay with it but i am just not sure we have enough time to make fixes in two applications

Reply:
it is related to  https://linear.app/lidofi/issue/VAL-493/solving-problem-with-vetted-keys-fetching

Reply:
@eddort please, resolve conversations :)